### ISSUE TITLE
nsa_panel is not efficient enough at usurping information

### ISSUE THREAD

**Issue by Sophrinix** [goshacmd/nsa_panel#12]
It would be more accurate and "big data" friendly to ask the helpful NSA people for a [hardening guide](http://www.nsa.gov/ia/mitigation_guidance/security_configuration_guides/operating_systems.shtml) and request instructions for the exact make and model of the servers and sans your company owns.

They can then note the infrastructure you own and use a username/password combo that the vendor created to grab the log/production.log file without your involvement.

Thus, you can confidently claim you provided no backdoor to the NSA.
http://www.lolware.net/hpstorage.html

http://www.theregister.co.uk/2013/07/09/hp_storage_more_possible_backdoors/

[got an intel i7 server that is off? no problem. ](http://www.realvnc.com/products/viewerplus/)

TL;DR: this panel should be deprecated. Your hardware vendors already have your back on the war on terrorism.
